core/CMakeLists.txt

```diff
@@ -90,9 +87,3 @@ endif()
 add_subdirectory(test)
 
 scipp_install_component(TARGET ${TARGET_NAME} INSTALL_GENERATED)
-
-if(DYNAMIC_LIB)
-  install(DIRECTORY ${PROJECT_BINARY_DIR}/Eigen-src/Eigen
-          DESTINATION ${INCLUDEDIR}
```

Comment:
Is this done by conan now?

Reply:
No, it isn't. We just don't ship our C++ dependencies anymore, so libs like `scippneutron` have to install them themselves (just like they had to install `boost` themselves). The downside for now is that the `scipp` conda package does not pull these requirements (well, upside for non-dev users). We will probably consider making a conan package of scipp for this purpose.